rpa-builder/6.0/modules/ROOT/pages/using-workflow-debugger.adoc

```diff
@@ -0,0 +1,57 @@
+
+image:workflow-debugger-window.png[The Workflow Debugger window]
+
+<1> The first pane in Workflow Debugger shows a condensed view of your Workflow and an arrow indicating which step was the last to execute.
+<2> On the second pane is the debug information for the selected Action Step. The Debug Information panel contains a combination of configuration settings and runtime information relating to Action Steps already executed.
+
+In this example, the Workflow entered the error state at the Window Wait Action Step. In the Debug Information panel, the *Result* section confirms that the window could not be found. Also, in the *Configuration* section, the *WindowTitle* value is set to `testname`, which causes the error because no window with that title exists during the execution.
+
+== Workflow Debugger Toolbar
+
+* (image:reload-workflow-icon.png[The Repeat icon, 2%, 2%]): *Repeat*
++
+Executes the Action Step again.
+* (image:edit-and-repeat-icon.png[The Edit & Repeat icon, 2%, 2%]): *Edit & Repeat*
++
+Opens the wizard for the last Action Step executed and enables you to modify its configuration and settings to fix the error. The Action Step then executes again to ensure it works correctly after the changes. The Workflow Debugger shows again on completion and you can check whether the *Run State* is _OK_ or _Error_.
+* (image:run-process.png[The Continue icon, 2%, 2%]): *Continue*
++
+Continues the Workflow execution.
++
+Note that the *Continue with Runstate:* drop-down menu reflects the current *Run State*. You can force the state to `Error` or `OK` at this point. For example, if you want to debug the _OnError_ part of a Managed Block, set the state to `Error` during the _DoAction_ section which subsequently leads to the execution of the _OnError_ section. You can also force the *Run State* to `OK` when you want the rest of the Workflow to be executed even though an error occurred.
+* (image:step-forward.png[The Single Step icon, 2%, 2%]): *Single Step*
++
+Use this button to execute the Workflow step-by-step.
+* (image:stop-icon.png[The Stop icon, 2%, 2%]): *Stop*
++
+Stops the current debugging run and returns to RPA Builder.
+
+Note that you can call the wizard for an Action Step at any time by double-clicking on it.
+
+== Using Breakpoints
+
+Breakpoints enable you to set points in the Workflow where the Workflow Debugger always stops the execution. You can either set breakpoints before starting a debug run or you can set them within the Workflow Debugger itself. Any Workflow object can have a breakpoint.
+
+To add a breakpoint to a Workflow object in the Workbench, right-click the object and select *Set Breakpoint* from the context menu. The breakpoint icon (image:breakpoint-icon.png[The Breakpoint icon, 2%, 2%]) shows next to that element.
```

Comment:
Just a general comment--it seems potentially tedious to have to manage all these icon images. Are they truly necessary?